service/src/main/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiController.java

```diff
@@ -47,7 +43,7 @@ public LandingZoneApiController(
 
   @Override
   public ResponseEntity<ApiCreateLandingZoneResult> createAzureLandingZone(
-      @RequestBody ApiCreateAzureLandingZoneRequestBody body) {
```

Comment:
all of these parameter annotations are handled by the `LandingZonesApi` interface and are not required. Some are explicitly forbidden by jakarta to be repeated